public/css/feature-tip-group.css

```diff
@@ -111,6 +111,10 @@
     max-width: 600px;
   }
 
+  .feature-tip-headline[data-lang="de"] {
+    word-break: break-word;
```

Comment:
Dumb question: what's the risk/effect of enabling this for all locales?

Reply:
So actually this is a great point and I think we can enable this for all locales. I was originally restricting this to German locales because I was using `word-break:break-all` which had the adverse affect of doing stuff like this: 

With `word-break:break-all;`...
<img width="321" alt="Screen Shot 2019-07-27 at 5 08 44 PM" src="https://user-images.githubusercontent.com/22355127/61999923-5be52080-b091-11e9-8275-4b9321576199.png">

As opposed to what happens with `word-break:break-word;` ...
<img width="321" alt="Screen Shot 2019-07-27 at 5 10 31 PM" src="https://user-images.githubusercontent.com/22355127/61999930-83d48400-b091-11e9-8c0b-09cc82b2c379.png">

So I think we're good to enable this for all locales for now and sanity check in Heroku. I'll update this PR.